test/functional/error_tests.js

```diff
@@ -270,8 +270,7 @@ describe.skip('Errors', function() {
     test: function(done) {
       var client = null;
 
-      process.once('uncaughtException', function(err) {
-        console.log(err);
+      process.once('uncaughtException', function() {
```

Comment:
Not necessary for this PR, but can we revisit this test? This seems like it is doing something wrong.